### PR TITLE
New Rules using org_sld lists

### PR DESCRIPTION
**Pull request by morriscode**

### CODE REVIEW

File: detection-rules/impersonation_org_sld_in_filename_fts.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: "Impersonation: Organizational SLD found in attachment file name from FTS"
+type: "query"
```
Comment (cameron-dunn-sublime) [inline detection-rules/impersonation_org_sld_in_filename_fts.yml:2]:
```suggestion
type: "rule"
```

File: detection-rules/impersonation_org_sld_in_sender_sld_fts.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: "Impersonation: Organizational SLD found in Sender domain"
+type: "query"
```
Comment (cameron-dunn-sublime) [inline detection-rules/impersonation_org_sld_in_sender_sld_fts.yml:2]:
```suggestion
type: "rule"
```

**Comment by jkamdjou** [comment]
should we just move these to signals?

**Pull request closed without merging**